doc/lua_api.txt

```diff
@@ -5286,8 +5271,8 @@ Object Properties
     --    Defaults to 'true'
     }
 
-Entity definition (`register_entity`)
--------------------------------------
+Entity definition
+-----------------
 
```

Comment:
Used by ("Passed to"?)  `minetest.register_entity`

Reply:
Used "Passed to" further down to mean passed to by the engine

Reply:
Ah yes, never mind... :)